editors/code/src/toolchain.ts

```diff
@@ -3,7 +3,7 @@ import * as os from 'os';
 import * as path from 'path';
 import * as readline from 'readline';
 import * as vscode from 'vscode';
-import { execute, log, memoize } from './util';
+import { execute, log, memoize, memoizeAsync } from './util';
```

Comment:
Remove the nonexistent `memoize` import. I'm not sure how this didn't get caught by tests. (?)